modules/central-configuration-options-operator.adoc

```diff
@@ -63,7 +63,7 @@ Otherwise, you must reinstall the custom resource to log back in.
 
 |`central.persistence.persistentVolumeClaim.claimName`
 |The name of the PVC to manage persistent data.
-If no PVC with the given name exists, it will be created. The default value is `stackrox-db` if not set. To prevent data losses the PVC is not removed automatically with Central`s deletion.
+If no PVC with the given name exists, it will be created. The default value is `stackrox-db` if not set. To prevent data loss, the PVC is not removed automatically with Central`s deletion.
```

Comment:
I think this is outdated, no? I believe it's called `central-db` nowadays.

Reply:
I opened https://issues.redhat.com/browse/ROX-23213 to investigate & make changes to the doc globally.